modules/developers-guide/pages/tutorials/hello-location.adoc

```diff
@@ -72,14 +73,14 @@ include::example$location_hello.mo[]
 ----
 . Save your changes and close the file to continue.
 
-== Start the local network
+== Start the local canister execution environment
 
-Before you can build your project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+Before you can build your project, you need to connect to a local canister execution environment or access a real subnet on the Internet Computer.
```

Comment:
```suggestion
Before you can build your project, you need to connect to a local canister execution environment or the Internet Computer blockchain mainnet.
```

Reply:
I should have noticed that in the Google Doc as well. The user does not really see a "subnet" in the deployment story – I think it is rather confusing than helpful here.